src/completions.jl

```diff
@@ -42,6 +48,20 @@ function basecompletionadapter(line, mod, force)
       continue
     end
   end
+  
+  # completions from the local code block:
+  for c in locals(text, lineNumber, column)
+    if startswith(c, pre)
+      pushfirst!(d, Dict(
+                  :type        => "",
+                  :icon        => "l",
```

Comment:
How about this iconography ?:
![image](https://user-images.githubusercontent.com/40514306/64125347-9a829f00-cde4-11e9-9ee7-b1e1c99e6dce.png)

Reply:
I went with `icon-chevron-right` instead, didn't like the magnifying glass :D

Reply:
Okay !